certified-connectors/Revelation helpdesk/apiProperties.json

```diff
@@ -0,0 +1,68 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "revSiteHost": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "displayName": "Revelation URL",
+          "description": "Your Revelation site URL (Example: https://support.yourdomain.com)",
+          "tooltip": "Provide your Revelation helpdesk site URL. This must be a secure Url (https://) and must NOT end with '/'"
+        }
+      },
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Revelation.PowerAutomate",
```

Comment:
@MVliegenthart Can you please replace it with `"[[DUMMY]]"`

Reply:
@sriyen-msft this is done.